build.yaml

```diff
@@ -11,6 +11,9 @@ targets:
       realm_generator|realm_object_builder:
         generate_for:
           include:
+            - test/**.dart
+            - example/**.dart
```

Comment:
Why do we need to include test and example folder?

Reply:
This is more to ensure that only test and example folders are processed.

Reply:
What about "flutter\realm_flutter\example"? It should be also included, I think.

Reply:
That is a project of its own with its own pubspec.yaml and buiild.yaml. You should run the generator in the directory.

Reply:
Yes, right.